config/locales/en.yml

```diff
@@ -234,6 +236,16 @@ en:
               blank: You must provide a provider name
             dttp_id:
               invalid: You must enter a DTTP ID in the correct format, like b77c821a-c12a-4133-8036-6ef1db146f9e
+        user:
+          attributes:
+            first_name: 
+              blank: "You must enter your first name"
```

Comment:
Since it won't be the actual users signing up themselves, we should probably return an error like:

`"You must enter the user's first name"`

(rather than 'your').]

The same goes for the translations for last name and email.

Reply:
thanks, fixed